### PR TITLE
Fix: Increase test coverage

### PR DESCRIPTION
**Pull request by dearchap**
## What type of PR is this?

_(REQUIRED)_

- cleanup

## What this PR does / why we need it:

_(REQUIRED)_

Try to get test coverage to above 90%

## Which issue(s) this PR fixes:

_(REQUIRED)_

## Special notes for your reviewer:

_(fill-in or delete this section)_

## Testing

go test

_(fill-in or delete this section)_

## Release Notes

_(REQUIRED)_

```release-note
No user facing changes for the most part. Mainly for increasing test coverage
```

### CODE REVIEW

File: flag.go [hunk 1]
```diff
@@ -73,26 +73,6 @@ var FlagEnvHinter FlagEnvHintFunc = withEnvHint
 // details. This is used by the default FlagStringer.
 var FlagFileHinter FlagFileHintFunc = withFileHint
 
-// FlagsByName is a slice of Flag.
-type FlagsByName []Flag
```
Comment (Juneezee) [inline flag.go:77]: CI error is from here https://github.com/urfave/cli/actions/runs/11450130089/job/31857025546?pr=1988#step:7:111

https://github.com/urfave/cli/blob/5053ec708a44d766e62f7e33baae742b6e2c003c/docs/v3/examples/flags.md?plain=1#L291-L363

This example is using `cli.FlagsByName`.

**Review by bartekpacia — APPROVED** [review]
more tests ❤️ thank you!

**Comment by tarampampam** [comment]
This PR broke the docs package. 🙂

Related issue: https://github.com/urfave/cli-docs/issues/5

Will the removal of `GetValue()` be reverted, or should I update the `cli-docs` package to work around it?

**Comment by dearchap** [comment]
yes i will revert GetValue

**Pull request merged**